docs/standard/data/sqlite/async.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Async Limitations - Microsoft.Data.Sqlite
```

Comment:
I've noticed you're missing the required SEO metadata called description in these articles.

https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master

Reply:
Updated.